docs/user/cassandra/commandline_tools.rst

```diff
@@ -0,0 +1,134 @@
+Cassandra Command Line Tools
+============================
+
+The GeoMesa Cassandra distribution comes with a set of command line tools
+for working with GeoMesa and Cassandra. This section shows how to use
+the tools.
+
+Before starting, check the following:
+
+- Make sure Cassandra is running (check by running ``bin/nodetool status`` in your
+  Cassandra installation directory.)
+- Make sure you've set the ``CASSANDRA_LIB`` environment variable (check by running
+  ``echo $CASSANDRA_LIB``)
+- Make sure you've created a key space within Cassandra (check by starting ``bin/cqlsh`` in your
+  Cassandra installation directory, and then by typing ``DESCRIBE KEYSPACES``)
+
+Then, ``cd`` into the ``GEOMESA_CASSANDRA_HOME`` directory, and type ``bin/geomesa-cassandra help``. You should
+get a listing of the tools with descriptions.
+
+The first tool we'll try is the ``ingest`` command. This command takes data from an external source, and
+ingests it into the Cassandra database. For this example, we'll use some sample data located
+at ``examples/ingest/csv/example.csv`` in the ``GEOMESA_CASSANDRA_HOME`` directory. Also, to make
+editing and running the command easier, we'll use the example bash script at ``examples/ingest/csv/ingest_csv.sh``.
+This is the script::
+
+    #!/usr/bin/env bash
+
+    # spec and converter for example_csv are registered in $GEOMESA_CASSANDRA_HOME/conf/application.conf
+    bin/geomesa-cassandra ingest \
+        --contact-point 127.0.0.1:9042 \
+        --key-space mykeyspace \
+        --catalog mycatalog \
+        --name-space mynamespace \
+        --converter example_csv \
+        --spec example_csv \
+        examples/ingest/csv/example.csv
+
+- The ``contact-point`` is the address a Cassandra node. If you're running Cassandra locally with
+  the default configuration, you shouldn't have to change this.
+- The ``key-space`` is the Cassandra key space that you created as part of configuring Cassandra (see above).
+  Edit this argument if needed to match the name of your key space.
+- The ``catalog`` argument specifies the name of a Cassandra table that will be created in your name space
+  by the ``ingest`` command.
+  You can name the catalog table anything you want.
+  This table is a metadata table that contains a list of each GeoMesa data table
+  that you have created. Before you run the ``ingest`` command for the first time, this table does
+  not exist. When you run ``ingest`` for the first time, GeoMesa creates two tables: this "catalog" metadata
+  table, and the table that actually stores your data. If you run ``ingest`` again with a different dataset,
+  GeoMesa will append a row to the "catalog" table, and also create an additional table for the new data.
+- The ``name-space`` parameter can be anything you want. (It is used by GeoTools to provide a name space for your feature types.)
```

Comment:
Agreed, but this is not a misspelling, the option passed to ``geomesa-cassandra`` actually *is* ``--name-space``. We could change the tool's options, but IMHO that should be a separate PR.